site/content/integrate/admin-guide/admin-zapier-integration.md

```diff
@@ -0,0 +1,148 @@
+
+**GitHub Issues and Comments, UserVoice Suggestions and Comments, GitLab MM Issues, GitLab Omnibus MM Issues**
+
+```
+    #### [Title of issue]
+
+    #[searchable-hashtag] in [external service](link to service) by [author](link to author profile) on [time-stamp](link to specific issue or comment)
+
+    [Body of issue or comment]
+```
+
+.. image:: ../images/zapier-ch1.png
+
+**Forum Posts, Jira Comments, Hacker News Mentions, Tweets**
+
+```
+     > [forum post, media mention, or tweet]
+
+     #[searchable-hashtag] in [external service](link to service) by [author](link to author profile) on [time-stamp](link to specific forum post, media mention or tweet)
+```
+
+.. image:: ../images/zapier-ch2.png
+
+## Troubleshooting Guide
+
+Possible solutions to common issues encountered during setup.
+
+### Cannot connect a Mattermost account
+
+1. `"Token named access_token was not found in oauth response!"`
+  a. Possible Solution: Try removing any trailing `/`'s on the end of your **Mattermost URL**.
+    - Correct: `https://community.mattermost.com`
+    - Incorrect: `https://community.mattermost.com/`
+
+  .. image:: ../images/zapier-error1.png
+
+2. `"[Server URL] returned (404)"`
+  a. Possible Solution: The **Mattermost URL** cannot have a team appended to the end of the server URL.
+    - Correct: `https://community.mattermost.com`
+    - Incorrect: `https://community.mattermost.com/core`
+
+  .. image:: ../images/zapier-error2.png
+
+3. `"[Server URL] returned (500) Internal Server Error"`
+  a. Possible Solution: The **Client Secret** might be incorrect. Verify this value in **Main Menu > Integrations > OAuth 2.0 Applications**, or check with your System Admin.
+
+  .. image:: ../images/zapier-error4.png
+
+4. `"Error Invalid client id"`
+  a. Possible Solution: The **Client ID** and/or **Client Secret** might have trailing spaces in them when copied and pasted into the form. Verify there are no trailing spaces in the **Client ID** and **Client Secret** fields then try again.
+
+  .. image:: ../images/zapier-trailing-space-error.png
+
+5. `"Mattermost needs your help: We couldn't find the requested app"`
+  a. Possible Solution: The **Client ID** might be incorrect. Verify this value in **Main Menu > Integrations > OAuth 2.0 Applications**, or check with your System Admin.
+
+  .. image:: ../images/zapier-error3.png
+
+### Deauthorize the Zapier App
+
+If you'd like to deauthorize Zapier so it can no longer post through your connected account, go to **Account Settings > Security > OAuth 2.0 Applications**, then click **Deauthorize** on the Zapier app.
```

Comment:
```suggestion
If you'd like to deauthorize Zapier so it can no longer post through your connected account, go to **Account Settings > Security > OAuth 2.0 Applications**, then select **Deauthorize** on the Zapier app.
```